requirements.txt

```diff
@@ -1 +1,2 @@
+markupsafe==2.0.1
```

Comment:
Pinned `markupsafe` as a solution to the issue that the CI check picked up:
https://app.circleci.com/pipelines/github/PennyLaneAI/pennylane.ai/313/workflows/f1dd81a4-b76d-4091-bb64-fc5723e66d50/jobs/368